tests/unit_test/cloud_utils_test.py

```
@@ -298,15 +303,19 @@ def test_trigger_lambda_model_training(self):
 
         def __mock_make_api_call(self, operation_name, kwargs):
             assert kwargs == {'FunctionName': 'train-model', 'InvocationType': 'RequestResponse', 'LogType': 'Tail',
-                              'Payload': b'{"BOT":"test","USER":"test_user"}'}
+                              'Payload': b'[{"name":"BOT","value":"test"},{"name":"USER","value":"test_user"}]'}
             if operation_name == 'Invoke':
                 return response
 
             raise Exception("Invalid operation_name")
 
         with patch.dict(Utility.environment, mock_env):
             with mock.patch('botocore.client.BaseClient._make_api_call', new=__mock_make_api_call):
```

Ruff

Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)